PWGMM/Mult/Tasks/dndeta-mft.cxx

```diff
@@ -74,49 +74,79 @@ struct PseudorapidityDensityMFT {
 
   Service<o2::framework::O2DatabasePDG> pdg;
 
-  Configurable<float> estimatorEta{"estimatorEta", 1.0, "eta range for INEL>0 sample definition"};
+  Configurable<float> estimatorEta{"estimatorEta", 1.0,
+                                   "eta range for INEL>0 sample definition"};
 
   Configurable<bool> useEvSel{"useEvSel", true, "use event selection"};
   ConfigurableAxis multBinning{"multBinning", {701, -0.5, 700.5}, ""};
 
   Configurable<bool> useZDiffCut{"useZDiffCut", true, "use Z difference cut"};
-  Configurable<float> maxZDiff{"maxZDiff", 1.0f, "max allowed Z difference for reconstruced collisions (cm)"};
+  Configurable<float> maxZDiff{
+      "maxZDiff", 1.0f,
+      "max allowed Z difference for reconstruced collisions (cm)"};
+
+  Configurable<float> cfgPhiCut{"cfgPhiCut", 0.1f,
+                                "Cut on azimuthal angle of MFT tracks"};
+
   //-----need access to CCDB to get the reweighting histogram
   Service<ccdb::BasicCCDBManager> ccdb;
-  Configurable<std::string> path{"ccdb-path", "Users/s/sherrman/My/Object", "base path to the ccdb object"};
-  Configurable<std::string> url{"ccdb-url", "http://alice-ccdb.cern.ch", "url of the ccdb repository"};
-  Configurable<int64_t> nolaterthan{"ccdb-no-later-than", std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count(), "latest acceptable timestamp of creation for the object"};
+  Configurable<std::string> path{"ccdb-path", "Users/s/sherrman/My/Object",
```

Comment:
These CCDB-related configurables are not used in the task, as I understand this is a leftover for the vertex Z re-weighting that was done at some point. You can remove them.

Reply:
Thanks for noticing, Anton. I removed them.